skt/executable.py

```diff
@@ -44,6 +45,58 @@ def full_path(path):
     return os.path.abspath(os.path.expanduser(path))
 
 
+def destroy_state_file(cfg):
+    """
+    Destroy the state file
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        os.unlink(state_file)
+
+
+def read_state_file(cfg):
+    """
+    Read the state file from the disk
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        with open(state_file, 'r') as fileh:
+            current_state = yaml.load(fileh)
+    else:
+        current_state = {}
+
+    return current_state
+
+
+def save_state_file(cfg, state_updates):
```

Comment:
Could you perhaps rename these functions so that it's clearer they're operating on the same stuff? E.g. name them `state_file_destroy`, `state_file_read`, and `state_file_save`? Or even make a module for them called `state_file.py` and name the functions `destroy`, `read`, and `save` respectively?

Oh, and also, `save_state_file` name is somewhat confusing and would imply that we're overwriting the file and not merging the data into it. I think something like `update_state_file` or `state_file_update` or `state_file.update` would be better.

Reply:
Makes sense. I'll work on that.